### PR TITLE
added immutable x docs

### PR DESCRIPTION
**Pull request by Saviour1001**

### CODE REVIEW

**Comment by shahbaz17** [comment]
Screenshot 2022-11-14 at 11 04 40 AM

ImmutableX logo is missing here.

File: docs/connect-blockchain/immutablex.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+const walletConnection = { ethSigner, starkSigner };
+const response = await client.registerOffchain(walletConnection);
+```
+
+## Get Balance
+
+---
+
+We can get the Layer 1 ( Ethereum ) balance using the below code snippet.
```
Comment (shahbaz17) [inline docs/connect-blockchain/immutablex.mdx:101]: @Saviour1001 Is there a way to get the balance of Layer 2?

File: src/components/GuidesMaps/index.ts [hunk 2]
```diff
@@ -348,6 +348,17 @@ export const referenceMap = [
     order: 1,
     link: "/docs/connect-blockchain/algorand",
   },
+  {
+    title: "Integrate Web3Auth with the ImmutableX Blockchain",
+    description: "Learn how to add a non custodial social login to the ImmutableX Blockchain",
+    image: "/docs/contents/guides/banners/immutablex.png",
```
Comment (shahbaz17) [inline src/components/GuidesMaps/index.ts:354]: Please add this banner here.

File: src/components/GuidesMaps/index.ts [hunk 3]
```diff
@@ -348,6 +348,17 @@ export const referenceMap = [
...
+    type: "REFERENCE",
+    tags: ["immutablex", "@web3auth/modal", "@web3auth/core"],
+    date: "3rd Oct 2022",
+    author: "Web3Auth Team",
+    order: 1,
```
Comment (shahbaz17) [inline src/components/GuidesMaps/index.ts:359]: Can you change the order here?

File: src/components/GuidesMaps/index.ts [hunk 4]
```diff
@@ -348,6 +348,17 @@ export const referenceMap = [
     order: 1,
     link: "/docs/connect-blockchain/algorand",
   },
+  {
+    title: "Integrate Web3Auth with the ImmutableX Blockchain",
+    description: "Learn how to add a non custodial social login to the ImmutableX Blockchain",
+    image: "/docs/contents/guides/banners/immutablex.png",
+    type: "REFERENCE",
+    tags: ["immutablex", "@web3auth/modal", "@web3auth/core"],
+    date: "3rd Oct 2022",
```
Comment (shahbaz17) [inline src/components/GuidesMaps/index.ts:357]: Date as well.

File: docs/connect-blockchain/immutablex.mdx [hunk 5]
```diff
@@ -0,0 +1,127 @@
+---
+title: Integrate Web3Auth with the ImmutableX Blockchain
+sidebar_label: ImmutableX
+image: "/docs/contents/guides/banners/algorand.png"
```
Comment (shahbaz17) [inline docs/connect-blockchain/immutablex.mdx:4]: Please update the image here with the immutable one.

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

File: docs/connect-blockchain/immutablex.mdx (hunk 1 above)
Comment (Saviour1001) [reply docs/connect-blockchain/immutablex.mdx:101]: There are withdrawal methods, but no helper function to get the balance in the immutable core SDK

File: src/components/GuidesMaps/index.ts (hunk 2 above)
Comment (Saviour1001) [reply src/components/GuidesMaps/index.ts:354]: done

File: src/components/GuidesMaps/index.ts (hunk 4 above)
Comment (Saviour1001) [reply src/components/GuidesMaps/index.ts]: Done

File: docs/connect-blockchain/immutablex.mdx (hunk 5 above)
Comment (Saviour1001) [reply docs/connect-blockchain/immutablex.mdx]: done

File: src/components/GuidesMaps/index.ts (hunk 3 above)
Comment (shahbaz17) [reply src/components/GuidesMaps/index.ts:359]: This is for reference. Got it.

**Review by shahbaz17 — APPROVED** [review]
LGTM

**Pull request merged**